evm.md

```diff
@@ -1326,6 +1325,13 @@ The various `CALL*` (and other inter-contract control flow) operations will be d
          <program> _ => BYTES </program>
          <jumpDests> _ => #computeValidJumpDests(BYTES) </jumpDests>
 
+    syntax KItem ::= "#touchAccounts" Account | "#touchAccounts" Account Account
+ // ----------------------------------------------------------------------------
+    rule <k> #touchAccounts ADDR1 ADDR2 => #touchAccounts ADDR1 ~> #touchAccounts ADDR2 ... </k>
+
+    rule <k> #touchAccounts ADDR => . ... </k>
+         <touchedAccounts> TOUCHED_ACCOUNTS => TOUCHED_ACCOUNTS |Set SetItem(ADDR) </touchedAccounts>
```

Comment:
Not really related to your changes, but what does the `touchedAccounts` cell represent?

Reply:
The Firefly client needs to keep track of the `<touchedAccounts>` (which are accounts that this execution affects, either by reading or writing from their state), in order to correctly report that to the testing client (eg. TruffleSuite).

Doesn't affect KEVM, except taht these rules are not working for the haskell backend because they use set concat instead of set union.